### PR TITLE
*: fix errcheck

### PR DESCRIPTION
**Pull request by jianzhiyao**
### What problem does this PR solve?

Issue Number: part of #22979

### What is changed and how it works?

What's Changed: added errcheck

### Check List 

Tests 

- No code

### Release note 

- No release note.

### CODE REVIEW

File: store/tikv/config/config_test.go [hunk 1]
```diff
@@ -34,20 +34,26 @@ func (s *testConfigSuite) TestParsePath(c *C) {
 }
 
 func (s *testConfigSuite) TestTxnScopeValue(c *C) {
-
-	failpoint.Enable("github.com/pingcap/tidb/store/tikv/config/injectTxnScope", `return("bj")`)
+	var err error
+	err = failpoint.Enable("github.com/pingcap/tidb/store/tikv/config/injectTxnScope", `return("bj")`)
```
Comment (wjhuang2016) [inline store/tikv/config/config_test.go:38]: How about c.Assert(failpoint.Enable("github.com/pingcap/tidb/store/tikv/config/injectTxnScope", `return("bj")`), IsNil)?

Comment (jianzhiyao) [reply store/tikv/config/config_test.go:38]: Nice idea

**Review by tisonkun — APPROVED** [review]
LGTM

**Review by wjhuang2016 — APPROVED** [review]

**Comment by tisonkun** [comment]
@wjhuang2016 could you please help trigger the merging phase?

**Pull request merged**